gulpfile.js

```diff
@@ -16,13 +16,15 @@ var info = require('./package.json');
 var request = require('request');
 
 var isWin = process.platform.indexOf('win') > -1;
+var isMac = process.platform.indexOf('darwin') > -1;
+var isLinux = process.platform.indexOf('linux') > -1;
```

Comment:
@jedahan Yes definitely. I was just following the logic for the respective files. Do you think the other logic is cleaner/clearer?